CONTRIBUTING.rst

```diff
@@ -0,0 +1,188 @@
+
+These range from very simple to moderately complex, but won't require you to understand the entire documentation or make extensive changes. We try to keep a few of these open for "micro contributions" for GSOD applicants and others looking for an easy-to-intermediate task. If you can't find one you like, ask `fcbond! <https://github.com/fcbond>`_
+
+**Submitting Pull Request (PR)**
+
+Google Season of Doc (GSOD) should submit their contributions to the `GSOD branch <https://github.com/globalwordnet/gwadoc/tree/gsod>`_ other contributors can submit to the `master branch <https://github.com/globalwordnet/gwadoc/pulls>`_
+
+Please submit a Pull Request (PR) with a clear list of what you have done. Also reference the issue number for which the PR fixes. Please follow the documentation style guide, PR template and make sure you write a commit message for each feature.
+
+Small pull requests are fine, but it's best if they can be "complete" in some sense. This means that if the pull request is for adding information about 1 or 2 relations, all the relevant information is included. Furthermore, the pull request should not bundle unrelated changes (e.g., a UI or settings change).
+
+The purpose of these guidelines is to make reviewing the pull request easier . It's easiest when we can clearly see all the relevant changes without other noise and to keep the Git history clean so later we can point to where something changed.
+
+Always write a clear log message for your commits. One-line messages are fine for small changes, but bigger changes should look like this:
+> git commit -m “A paragraph describing what changed and its impact”
+
+**Before opening a PR**
+
+Create a git branch for the issue you are working on, rather than working directly on master.
+Rebase your work onto the latest version from master and fix any merge conflicts. (See `README <https://github.com/globalwordnet/gwadoc/blob/master/README.md>`_ for common Git procedures.)
+
+Check that only the relevant files are being changed.
+If you have a messy git history, squash your commits to clean it up.
+
+**PR format**
+
+Title your pull request to indicate its purpose. (This will likely be similar to the title of the issue it addresses.)
+Include a reference to the issue(s).
+
+If it is a work-in-progress, include [WIP] in the title. Remove this once the PR is complete and ready for review.
+Include before-and-after screenshots (or animations) for user interface changes.
+
+**Documentation Convention**
+
+Our documentation project setup can be found in the `README <https://github.com/globalwordnet/gwadoc/blob/master/README.md>`_
+
+Start reading our documentation and you’ll get the hang of it. We optimize for readability
+We indent using two spaces
+Most of our contributions happen in the doc_en.py file and we generate the html file
+When writing consider your readers, as this is an open source software. The goal is to make the documentation readers friendly.
```

Comment:
This information and that below is more useful as it explains what is particular to this project. At some point it might make sense to put this in the main documentation pages, but for now this is a good spot.

Reply:
Thanks

Reply:
Content has been modified